_posts/2021-08-23-resteasy-reactive-smart-dispatch.adoc

```diff
@@ -0,0 +1,339 @@
+---
+layout: post
+title: RESTEasy Reactive - To block or not to block
+date: 2021-08-25   
+tags: reactive rest
+synopsis: Quarkus 2.2 includes a new dispatching strategy to make RESTEasy Reactive even better.
+author: cescoffier
+---
+
+:imagesdir: /assets/images/posts
+
+In January 2020, the Quarkus team announced RESTEasy Reactive, a novel way to serve HTTP API in Quarkus. 
```

Comment:
I think it was 2021, no?

Reply:
Wait a minute, aren't we in 2020? Damned!